auth/client/src/main/java/org/wildfly/security/auth/client/CredentialStoreFactory.java

```diff
@@ -81,10 +81,12 @@ public CredentialStore get() throws ConfigXMLParseException {
         try {
             if (providers != null) {
                 credentialStore = providerName != null ? CredentialStore.getInstance(type, providerName, providers) : CredentialStore.getInstance(type, providers);
+                credentialStore.initialize(attributes, credentialSource == null ? null : new CredentialStore.CredentialSourceProtectionParameter(credentialSource.get()), providers.get());
+
```

Comment:
Just a very minor comment, the extra line here can be removed.

Reply:
fixed!